src/test/java/com/powsybl/openloadflow/ac/AcEquationsTest.java

```diff
@@ -141,6 +142,28 @@ void branchTest() {
         assertArrayEquals(new double[] {0.3420075216110214, Double.NaN, Double.NaN, 0.31907275662806295, Double.NaN, Double.NaN, Double.NaN},
                 eval(new OpenBranchSide2CurrentMagnitudeEquationTerm(branch, bus2, variableSet, false, false), variables, sv));
 
+        // assert current equation is consistent with active and reactive power ones
+        var p1Eq = new ClosedBranchSide1ActiveFlowEquationTerm(branch, bus1, bus2, variableSet, true, true);
+        p1Eq.setStateVector(sv);
+        double p1 = p1Eq.eval();
+        var q1Eq = new ClosedBranchSide1ReactiveFlowEquationTerm(branch, bus1, bus2, variableSet, true, true);
+        q1Eq.setStateVector(sv);
+        double q1 = q1Eq.eval();
+        var i1Eq = new ClosedBranchSide1CurrentMagnitudeEquationTerm(branch, bus1, bus2, variableSet, true, true);
+        i1Eq.setStateVector(sv);
+        double i1 = i1Eq.eval();
+        assertEquals(i1, Math.hypot(p1, q1) / V_1, 10e-14);
```

Comment:
@annetill is it normal to not have the sqrt(3) here ?

Reply:
After looking at the code, yes it is normal. The sqrt(3) is managed by function PerUnit.ib() for sensitivity and security analysis.

Reply:
It doesn't mean it is correct. As far as I understand main principle of peruniting is to perunit only 2 quantities (for us it is apparent power and voltage) and all the other quantities are calculated from the 2 chosen ones.
So for me, the values in per unit are not correct even if one in SI are correct.

Reply:
According https://en.wikipedia.org/wiki/Per-unit_system it seems correct